sdk/test-utils/test-utils/package.json

```diff
@@ -56,22 +56,24 @@
   "sideEffects": false,
   "private": true,
   "dependencies": {
-    "mocha": "^7.1.1",
     "@azure-tools/test-recorder": "^1.0.0",
-    "tslib": "^2.2.0",
     "@azure/core-tracing": "1.0.0-preview.14",
-    "@opentelemetry/api": "^1.0.3"
+    "@opentelemetry/api": "^1.0.3",
+    "mocha": "^7.1.1",
+    "tslib": "^2.2.0"
   },
   "devDependencies": {
     "@azure/dev-tool": "^1.0.0",
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@microsoft/api-extractor": "^7.18.11",
     "@types/chai": "^4.1.6",
+    "@types/chai-as-promised": "^7.1.4",
     "@types/mocha": "^7.0.2",
     "@types/node": "^12.0.0",
     "@types/sinon": "^9.0.4",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
+    "chai-exclude": "^2.1.0",
```

Comment:
now that the package is re-exporting chai primitives, I think these should be moved to the `dependencies` section.

Reply:
Thank you Deya - I was debating what the right thing to do is (deps / devDeps), glad you pointed this out 😄 will fix shortly!

Reply:
updated, thanks! btw by moving @types/chai* to dependencies as well I no longer had to re-export the types for them. I still don't know what I'm doing but hey 🤷